toptal/htmlshell
# [SAT-445] Small copy change in UI Wrap

We had a small change in our copy for UI Wrap.

## How to test

* Run project
* Check if the current text is updated.
* Toptal.com needs to send user to our home page.
